docs-src/docs/guides/walkthrough.md

```diff
@@ -355,35 +399,432 @@ When you're finished with the example, you can instruct Terraform to destroy the
+- `demo-env`: the name of the environment in which the service is to reside
+- `redis-svc`: a name for the service we're creating
+- `redis-dpl:latest`: the name of the deploy the service will put into action
+    - _(We use `:` to specify which deploy we want - `:latest` will always give us the most recently-created one.)_
+
+
+---
+
+### Part 4: Check the Status of the Service
+
+As in the first deployment, we can keep an eye on our service by using the `service get` command:
+
+`l0 service get redis-svc`
+
+Once the service has finished scaling, try looking at the service's logs to see the output that the Redis server creates:
+
+`l0 service logs redis-svc`
+
+Among some warnings and information not important to this exercise and a fun bit of ASCII art, you should see something like the following:
+
+```
+... # words and ASCII art
+1:M 05 Apr 23:29:47.333 * The server is now ready to accept connections on port 6379
+```
+
+Now we just need to teach the Guestbook application how to talk with our Redis service.
+
+
+---
+
+### Part 5: Update the Guestbook Deploy
+
+You should see in `iterative-walkthrough/deployment-2/` another `Guestbook.Dockerrun.aws.json` file. This file is very similar to but not the same as the one in `deployment-1/` - if you open it up, you can see the following additions:
+
+```
+    ...
+    "environment": [
+        {
+            "name": "REDIS_ADDRESS_AND_PORT",
+            "value": "${redis_address}"
+        }
+    ],
+    ...
+```
+
+That `value` is what will point the Guestbook application towards the Redis server. The `${redis_address}` needs to be replaced and populated in the following format:
+
+```
+"value": "ADDRESS_TO_REDIS_SERVER:PORT_THE_SERVER_IS_SERVING_ON"
+```
+
+We already know that Redis is serving on port 6379, so let's go find the server's address. Remember, it lives behind a load balancer that we made, so run the following command:
+
+`l0 loadbalancer get redis-lb`
+
+We should see output like the following:
+
+```
+LOADBALANCER ID  LOADBALANCER NAME  ENVIRONMENT  SERVICE    PORTS          PUBLIC  URL
+redislb16ae6     redis-lb           demo-env     redis-svc  6379:6379/TCP  false   internal-l0-<yadda-yadda>.elb.amazonaws.com
+```
+
+Copy that `URL` value, replace `${redis_address}` with the `URL` value in `Guestbook.Dockerrun.aws.json`, append `:6379` to it, and save the file. It should look something like the following:
+
+```
+    ...
+    "environment": [
+        {
+            "name": "REDIS_ADDRESS_AND_PORT",
+            "value": "internal-l0-<yadda-yadda>.elb.amazonaws.com:6379"
+        }
+    ],
+    ...
+```
+
+Now, we can create an updated deploy:
+
+`l0 deploy create Guestbook.Dockerrun.aws.json guestbook-dpl`
+
+We should see output like the following:
+
+```
+DEPLOY ID        DEPLOY NAME    VERSION
+guestbook-dpl.2  guestbook-dpl  2
+```
+
+
+---
+
+### Part 6: Update the Service
```

Comment:
also in this section, should be sure to qualify the `guestbook` service vs the `redis` service.